eskip/eskip.go

```diff
@@ -85,6 +109,10 @@ type Route struct {
 	// E.g. HeaderRegexp("Accept", /\Wapplication\/json\W/)
 	HeaderRegexps map[string][]string
 
+	// Custom predicates to match.
+	// E.g. Traffic(.3)
+	CustomPredicates []*CustomPredicate
```

Comment:
To expand on my previous suggestion, this would only be `Predicates []*Predicate` Seems cleaner to me